modules/sw360/sw360-workflow/src/main/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdater.java

```diff
@@ -142,6 +147,24 @@ public SW360Release getOrCreateRelease(SW360Release sw360ReleaseFromArtifact, bo
         return releaseClientAdapter.createRelease(sw360ReleaseFromArtifact);
     }
 
+    /**
+     * Checks whether a given release has a higher or equal ordered clearing release state than a given clearing state string.
+     * @param sw360ReleaseFromArtifact given release that might have a clearing state information
+     * @param clearingState Optional clearing state that gets checked against
+     * @return true if the release as a higher or equal clearing state to the optional string or if the optional clearing state or both are null or empty
+     */
+    private boolean artifactClearingStateIsHigherOrEqualToSW360Release(SW360Release sw360ReleaseFromArtifact, Optional<String> clearingState) {
```

Comment:
Again a rather technical method name. Something like _canUpdate()_ or _shouldUpdate()_ would reveal the intention clearer.